faucetagent.py

```diff
@@ -355,11 +356,12 @@ def main():
     # FaucetAgent handles gNMI requests
     agent = FaucetAgent(proxy)
     # Start the FAUCET gNMI service
-    info('Starting FAUCET gNMI configuration agent on port %d', args.gnmiport)
+    gnmi_url = '{0}:{1}'.format(args.gnmiaddr, args.gnmiport)
+    info('Starting FAUCET gNMI configuration agent on port %s', gnmi_url)
```

Comment:
I think "port" here is inaccurate.

Reply:
Should I just remove port so it just says "agent on ..." ?

Reply:
Works for me.